packages/core/components/Modal/EditMetadata/index.tsx

```diff
@@ -18,16 +18,23 @@ import styles from "./EditMetadata.module.css";
 // a more robust solution in the future such as login or a more secure method.
 // or find a way to get this information passed in via GH secrets to the packaged
 // web bundle
+// TODO: Actually lets try to request this from GH or something..?
```

Comment:
TODO: Before merging - test putting this into a file or something at least to get it out of this level of hard-coded

Reply:
I moved to the bucket (ty Anya)